securityhub-org/sumologic-securityhub-collector-awsorg/sam/event.json

```diff
@@ -0,0 +1,54 @@
+{
+    "account": "956882702234",
```

Comment:
there is already a testevent.json file in test folder Is this file different than the one already present?